gradle/scala.gradle

```diff
@@ -49,12 +49,18 @@ test {
 sourceSets {
     main {
         scala {
-            srcDir "src/main/scala_$scalaBaseVersion"
+            srcDirs += ["src/main/scala_$scalaBaseVersion", "src/main/java"]
+        }
+        java {
+            srcDirs = []
         }
     }
```

Comment:
Good point regarding dependencies!

`srcDir` expects one resource dir and add it to the existing resource directories
`srcDirs = ` sets new set of source directories
`srcDirs += ` should add to the existing source directories

At least from what I know